src/wincameranodelet.cpp

```diff
@@ -0,0 +1,121 @@
+#include "wincapture.h"
+#include "winrospublisher.h"
+#include <ros/ros.h>
+#include <string>
+#include <nodelet/nodelet.h>
+
+namespace ros_win_camera
+{
+    const int32_t PUBLISHER_BUFFER_SIZE = 4;
+
+    class WinCameraNodelet : public nodelet::Nodelet
+    {
+
+    public:
+        WinCameraNodelet()
+        {
+        }
+        ~WinCameraNodelet()
+        {
+            waitForFinish.lock();
+        }
+
+    private:
+        virtual void onInit()//int main(int argc, char** argv)
+        {
+            //ros::init(argc, argv, "win_camera");

```

Comment:
Remove commented code

Reply:
Forgot to mention in the description... I am still working on the nodelet code. This code will change significantly in my updates to come..

---
In reply to: [383532875](https://github.com/ms-iot/ros_win_camera/pull/2#discussion_r383532875) [](ancestors = 383532875)